test/paprController/PurchaseLiquidationAuctionNFT.t.sol

```diff
@@ -316,33 +316,6 @@ contract PurchaseLiquidationAuctionNFT is BasePaprControllerTest {
         controller.purchaseLiquidationAuctionNFT(auction, auction.startPrice, purchaser, oracleInfo);
     }
 
-    /// @dev we want to prevent abuse of our clearing remaining debt when there is a shortfall
-    /// but it is the borrowers last NFT
-    function testReentrancyOnLastNFTDoesNotClearDebt() public {
```

Comment:
why did we have to remove this test? probably still want to test the vulnerability doesn't exist?